Tasks/FtpUploadV2/task.json

```diff
@@ -18,7 +18,7 @@
     "demands": [],
     "version": {
         "Major": 2,
-        "Minor": 154,
+        "Minor": 157,
```

Comment:
When is the cutoff for this? Is this going to make it into 157 or should it be bumped to 158?

Reply:
I plan to backport this change to m157 later.  The branch is already cut.  If we do m158, it will take too long to go out.